src/Agent.Listener/MessageListener.cs

```diff
@@ -103,6 +103,7 @@ public override void Initialize(IHostContext hostContext)
                     if (ex is TaskAgentNotFoundException)
                     {
                         Trace.Error("The agent no longer exists on the server. Stopping the agent.");
+                        _term.WriteError(StringUtil.Loc("MissingAgent"));
```

Comment:
added special message for TaskAgentSessionConflictException. The other fatal exceptions are printing a generic error message.